src/index.js

```diff
@@ -1 +1,8 @@
+import loadArticles from './loadArticles.js';
 import './style.css';
+
+const shows = 'https://api.tvmaze.com/shows';
+
+fetch(shows)
+  .then((response) => response.json())
+  .then((data) => loadArticles(data));
```

Comment:
I personally think is better to create a function to get the data so that we dont have to run this lines every time we require the data 

I suggest doing something like this 

`const getData = async () => {
  const response =  await fetch(shows);
  const data = await response.json;
  return data 
}`